mobify/platform-scaffold
# Feature/mbfycnct 41 routing issues

Changed product and category routes to */c/{cid} and */p/{pid} to match Hybris OOTB routes

 **JIRA**: [MBFYCNCT-41](https://thinkwrap.jira.com/browse/MBFYCNCT-41)

## How to test-drive this PR
- First of all, run `npm run update-loader-routes`
- Navigate through the category tree and check that the routes have the form http://example.com/c/* and work
- Navigate to products and check that the routes have the form http://example.com/p/* and work
- Add to a category url some stuff before the /c/ and check that it works: http://example.com/some/stuff/c/*
- From here, continue navigating and check that it works
- Add to a product url some stuff before the /p/ and check that it works: http://example.com/some/stuff/p/*
- Change variants and check that it works
